src/backend/InvenTree/common/apps.py

```diff
@@ -40,28 +35,5 @@ def clear_restart_flag(self):
 
                 if not InvenTree.ready.isImportingData():
                     set_global_setting('SERVER_RESTART_REQUIRED', False, None)
-        except Exception:
+        except Exception:  # pragma: no cover
             pass
-
-    def override_global_settings(self):
-        """Update global settings based on environment variables."""
-        overrides = global_setting_overrides()
-
-        if not overrides:
-            return
-
-        for key, value in overrides.items():
-            try:
-                current_value = get_global_setting(key, create=False)
-
-                if current_value != value:
-                    logger.info(
-                        'INVE-I1: Overriding global setting: %s = %s',
-                        value,
-                        current_value,
-                    )
-                    set_global_setting(key, value, None, create=True)
```

Comment:
@SchrodingersGat this part of the code is never reachable, as the value returned `get_global_setting` will always be overridden by the override value it is compared to- there can be no difference at this point. Therefore, I have removed the whole section

Reply:
Ok, thanks for the catch